packages/payment-integration-test-framework/src/fixture/pageObject/playwright/PollyObject.ts

```diff
@@ -92,7 +92,7 @@ export class PollyObject {
     }
 
     stop(): Promise<void> | undefined {
-        return this.polly && this.polly.stop();
```

Comment:
We actually don't need this `stop()` to return anything....The current `Promise<void> | undefined` is not helpful either. Can we align this with the `play()`?
```
    play(): void {
        this.polly?.play();
    }
```